src/AppBundle/Mail/Sender.php

```diff
@@ -5,45 +5,66 @@
  */
 namespace AppBundle\Mail;
 
-use AppBundle\Entity\Message;
+use AppBundle\Model\Contact;
+use Symfony\Bundle\TwigBundle\TwigEngine as Templating;
 use Swift_Mailer;
 use Swift_Message;
+use Symfony\Component\Translation\TranslatorInterface;
 
 class Sender
 {
     /** @var \Swift_Mailer */
     protected $mailer;
 
+    /** @var  \Symfony\Component\Translation\TranslatorInterface */
+    protected $translator;
+
+    /** @var \Symfony\Bundle\TwigBundle\TwigEngine */
+    protected $templating;
+
+    /** @var string */
+    protected $senderEmail;
+
+    /** @var string */
+    protected $recipientEmail;
+
     /**
      * @param \Swift_Mailer $mailer
+     * @param \Symfony\Component\Translation\TranslatorInterface $translator
+     * @param \Symfony\Bundle\TwigBundle\TwigEngine $templating
+     * @param string $senderEmail
+     * @param string $recipientEmail
      */
-    public function __construct(Swift_Mailer $mailer)
-    {
+    public function __construct(
+        Swift_Mailer $mailer,
+        TranslatorInterface $translator,
+        Templating $templating,
+        $senderEmail,
+        $recipientEmail
+    ) {
         $this->mailer = $mailer;
+        $this->translator = $translator;
+        $this->templating = $templating;
+        $this->senderEmail = $senderEmail;
+        $this->recipientEmail = $recipientEmail;
     }
 
     /**
-     * Sends e-mail with content based on form settings.
-     *
-     * @param \AppBundle\Entity\Message $message
-     * @param string $title
-     * @param string $senderEmail
-     * @param string $recipientEmail
-     *
-     * @return bool
+     * @param Contact $contact
      */
-    public function send(Message $message, $title, $senderEmail, $recipientEmail)
+    public function send(Contact $contact)
     {
-        $msg = Swift_Message::newInstance($title, $message->getBody());
-
-        $msg->setFrom($senderEmail)
-            ->setTo($recipientEmail)
-            ->setReplyTo($recipientEmail);
+        $title = $this->translator->trans('You have a new message from ') . $contact->getFrom();
```

Comment:
I prefer to use `sprintf` for strings like this but if you want to refactor translations later then it's fine to leave it like that.

`sprintf('You have a new message from %s', $contact->getFrom())`gives more freedom to translators